windows/windows.py

```diff
@@ -0,0 +1,198 @@
+import subprocess
+import os
+import re
+import sys
+import zipfile
+
+sys.path.append("..")
+
+from utility.check_internet import check_internet
+from utility.install_package import install
+
+from path import path, var_dir, cred_dir
+
+def check_internet():
+    try:
+        subprocess.check_output(["ping", "www.google.com"])
+        return True
+    except subprocess.CalledProcessError:
+        return False
+
+def setup():
+    '''
+    This function is used to setup the script and install required dependencies when invoked for the first time on user's system.
+    '''
+    print("Checking dependencies...")
+
+    # Creating a variable to store whether the algorithm is connected for the first time
+    # var_dir = os.path.join(path, "var")
+    if os.path.exists(var_dir):
+        pass
+    else:
+        print(var_dir)
+        os.mkdir(var_dir)
+        os.chdir(var_dir)
+        var_file= open("var.txt","w")
+        var_file.write("Packages installed : False\nSelenium installed : False\nCryptography installed : False\nWget installed : False\nChrome driver installed : False\nChrome installed : False\nSafari driver installed : False\nSafari installed : False\nFirefox driver installed : False\nFirefox installed : False\nInternet connection : False\nSafari Driver Enabled: False")
+        var_file.close()
+        os.chdir(path)
+
+    def check_true_dependencies():
+        with open(f"{var_dir}/var.txt", "r") as f:
+            for line in f:
+                if line.find("False"):
+                    return False
+            return True
+            
+    if check_true_dependencies() == True:
+        pass
+    elif check_internet() == True:
```

Comment:
don't try to install all the dependencies, some dependencies are specific for different os, no need for them in windows